module/x/gravity/types/genesis.go

```diff
@@ -141,7 +141,7 @@ func DefaultParams() *Params {
 		SlashFractionConflictingEthereumSignature: sdk.NewDec(1).Quo(sdk.NewDec(1000)),
 		UnbondSlashingSignerSetTxsWindow:          10000,
 		// this is in ethereum blocks. ethereum block time is ~12 seconds, about twice as long as Sommelier
-		EventVoteWindow:           5000,
+		EthereumEventVoteWindow:           5000,
```

Comment:
Nit: formatting seems off here, also maybe move this comment to the end of the line so it's clear it only applies to this one param?